src/pages/Home.tsx

```diff
@@ -21,10 +25,137 @@ import {jsx, SxStyleProp} from 'theme-ui';
  * @returns The home page. 
  */
 export const Home: React.FC = () => {
+  //the board
+  const board = useContext<IBoardContext>(BoardContext);
+
+  
+  const [posts, setPosts] = useState<IPost[]>([]);
+
+  
+
+  // TODO: get data from server
+
+  /**
+   * This uses to `fetch` API to request more posts from the server.  
+   * This is used to ensure that the server can dynamically load posts upon request and
+   * not load in everything at once.
+   * 
+   * For example, if the user wishes to load 30 more posts after seeing 50 posts,
+   * this will be `getMorePosts(50, 30)`. 
+   * This is zero-based index with lower bound (the index) inclusive. 
```

Comment:
What is the lower bound? It sounds like you're not really specifying bounds here at all, but instead specifying an offset index and load value.

What is the bound/index for? Can you elaborate here and in the parameter documentation?